simuvex/plugins/fast_memory.py

```diff
@@ -159,10 +159,10 @@ def _store(self, req):
 
         if (self.category == 'mem' and options.SIMPLIFY_MEMORY_WRITES in self.state.options) or \
            (self.category == 'reg' and options.SIMPLIFY_REGISTER_WRITES in self.state.options):
-            req.simplified_values = [ self.state.se.simplify(data) ]
+            simplified_values = [ self.state.se.simplify(data) ]
         else:
-            req.simplified_values = [ data ]
-        data = req.simplified_values[0]
+            simplified_values = [ data ]
```

Comment:
Yes, I removed simplified_values as a variable from request, but missed that this can be easily simplified.